src/main/webapp/app/overview/courses.component.html

```diff
@@ -19,43 +19,51 @@ <h4 class="ms-2 fw-medium">{{ nextRelevantExam.title }}</h4>
-            <jhi-overview-course-card class="col-12 col-lg-6 col-xl-4 pe-2 ps-2 mb-3" [course]="course" [hasGuidedTour]="course === courseForGuidedTour" />
+<div class="module-bg rounded-3">
+    <div class="d-flex justify-content-between module-bg p-3 rounded-3 align-items-center">
+        <h3 class="fw-medium mb-0" jhiTranslate="artemisApp.studentDashboard.title"></h3>
+        @if (regularCourses.length) {
+            <div class="col-sm-auto d-flex">
+                <a class="btn btn-primary" [routerLink]="['/courses/enroll']" jhiTranslate="artemisApp.studentDashboard.enroll.title"></a>
+            </div>
         }
     </div>
-    @if (regularCourses.length > 0) {
+    @if (recentlyAccessedCourses.length) {
         <div class="row">
-            <h4 class="col mb-3 fw-medium" jhiTranslate="artemisApp.studentDashboard.otherCourses"></h4>
+            <h4 class="col m-3 fw-medium" jhiTranslate="artemisApp.studentDashboard.recentlyAccessed"></h4>
         </div>
-    }
-}
-@if (coursesLoaded && !regularCourses.length && !recentlyAccessedCourses.length) {
-    <div class="mt-5 mb-5 d-flex flex-column gap-3">
-        <h4 class="text-secondary text-center" jhiTranslate="artemisApp.studentDashboard.noCourses"></h4>
-        <div class="col-sm-auto d-flex justify-content-center">
-            <a class="btn btn-primary" [routerLink]="['/courses/enroll']" jhiTranslate="artemisApp.studentDashboard.enroll.title"></a>
+        <div class="row mx-2">
+            @for (course of recentlyAccessedCourses; track course) {
+                <ng-template *ngTemplateOutlet="courseCardTemplate; context: { $implicit: course }" />
+            }
         </div>
-    </div>
-} @else {
-    <div class="row">
-        @for (course of regularCourses; track course) {
-            <jhi-overview-course-card class="col-12 col-lg-6 col-xl-4 pe-2 ps-2 mb-3" [course]="course" [hasGuidedTour]="course === courseForGuidedTour" />
+        @if (regularCourses.length) {
+            <div class="row">
+                <h4 class="col m-3 fw-medium" jhiTranslate="artemisApp.studentDashboard.otherCourses"></h4>
+            </div>
         }
-    </div>
-}
+    }
+    @if (coursesLoaded && !regularCourses.length && !recentlyAccessedCourses.length) {
+        <div class="mt-5 mb-5 d-flex flex-column gap-3">
+            <h4 class="text-secondary text-center" jhiTranslate="artemisApp.studentDashboard.noCourses"></h4>
+            <div class="col-sm-auto d-flex justify-content-center">
+                <a class="btn btn-primary" [routerLink]="['/courses/enroll']" jhiTranslate="artemisApp.studentDashboard.enroll.title"></a>
+            </div>
+        </div>
+    } @else {
+        <div class="row mx-2">
+            @for (course of regularCourses; track course) {
+                <ng-template *ngTemplateOutlet="courseCardTemplate; context: { $implicit: course }" />
+            }
+        </div>
+    }
+</div>
+
+<!-- course card template -->
```

Comment:
I do not think that this comment adds a value, rather a comment for the future, does not need to be changed
```suggestion
```